whd/dispatcher.py

```diff
@@ -205,34 +205,66 @@ def _set_pr_labels(self, pr_event, resources) -> bool:
             name,
             github_cfg=github_cfg,
         )
+        sender_login = pr_event.sender()['login']
         if pr_event.action() is PullRequestAction.OPENED:
             if github_helper.is_pr_created_by_org_member(pr_number):
                 logger.info(
                     f"New pull request by member of '{owner}' in '{repository_path}' found. "
                     f"Setting required labels '{required_labels}'."
                 )
+                github_helper.add_comment_to_pr(
+                    pull_request_number=pr_number,
+                    comment=(
+                        f"Thank you {sender_login} for your contribution. I will start a build for "
```

Comment:
Not sure whether this'd work, but consider using @{sender_login}. (Also for all the other occurrences)